webview/src/setup/components/App.tsx

```diff
@@ -90,6 +94,10 @@ export const App: React.FC = () => {
     })
   }
 
+  const closeSection = (section: SetupSection) => {
+    setSectionCollapsed({ ...sectionCollapsed, [section]: true })
```

Comment:
Thoughts so far:

* Add completion status to steps first before adding any kind of auto-close logic. I feel like it could make things more clear.
* Don't just autoclose DVC, open other sections since neither can work until DVC is setup.
* Experiments needs to autoclose and open webview OR just leave as is. Not sure which is better yet.